node/src/chain_specs/manta/mainnet_genesis.rs

```diff
@@ -0,0 +1,194 @@
+use super::*;
+use hex_literal::hex;
+use sc_network::config::MultiaddrWithPeerId;
+use sc_telemetry::TelemetryEndpoints;
+use sp_core::crypto::UncheckedInto;
+#[derive(Clone)]
+struct Collator {
+    acc: AccountId,
+    nodeid: MultiaddrWithPeerId,
+    keys: SessionKeys,
+}
+impl Collator {
+    fn new(acc: AccountId, nodeid: MultiaddrWithPeerId, keys: SessionKeys) -> Collator {
+        Self { acc, nodeid, keys }
+    }
+}
+
+fn manta_mainnet_genesis(genesis_collators: Vec<Collator>) -> GenesisConfig {
```

Comment:
Looking for opinions @ghzlatarev @Dengjianping  on whether you guys think preserving this code is worth the subfolder structure.
We can split the module further to separate the various testnets as we go along to keep things tidy.

Reply:
Why not put this in the manta.rs file, not sure I understand,

Reply:
1. This is dead code that's unused after creating the genesis once
2. This is long and obscures the active code in there

Reply:
Oh I see now, I thought the whole manta.rs file was deleted. 
This seems ok to me. If someone needs to make a new chain-spec, they won't have to find this old PR and copy the file :D

Reply:
I'm ok with a new folder for manta chain spec.